src/main/java/gov/nasa/jpl/mbee/mdk/mms/validation/BranchValidator.java

```diff
@@ -186,6 +188,9 @@ else if (serverBranch == null) {
                     continue;
                 }
                 serverBranch.remove(MDKConstants.STATUS_KEY);
+                serverBranch.remove(MDKConstants.PARENT_COMMIT_ID);
+
+                clientBranch.getValue().put(MDKJsonConstants.DELETED_FIELD, false); // TODO we need to fix this
```

Comment:
This line assumes only non-deleted branches are returned from MMS. This was marked to revisit in case MMS returned deleted branches.  Since MMS does not return deleted branches, we can remove the TODO.